.travis.yml

```diff
@@ -1,40 +1,43 @@
-language: python
-python:
-- '3.5'
-install:
-- pip install .
-script:
-- python -m doctest -v README.md
+before_install:
+- pip3 install black
 deploy:
 - provider: releases
   skip_cleanup: true
   api_key:
     secure: QcZBC5FRMFhS+DyOR5IzKaOrT9P6fMXkEKtKIDdl/qG/qTKj8fmAGrO4cdQRjysEElLaOtO04xjyNBs5Z+uHLGuZv+w6/D31AnOdDrIuv+96dWRAjqhUubU2yQYBV8eYo22QILXarImBqmjHSv2LKPDapZqutariELpuagE7uMfgftbA1hASSReY9CdXnPvbt3tM2gvvtCFCy1w+AZtEt6uUJcymk8meU4PkL3fyCYXczdx/KTqWQGSOIm6zdEyjgxyaD7qzjqFBHBp6NCXedL0AgDcJ5JcH7nlOJ9LhAu03J8nENvfHwFhxi+GBolB6OQChxapdZhkfCtKHHwWCM7j3Z9LTeZdVHlwU63T49QZJRd8+fHaLKzrE+y746Qg25QqUjw3mwhuxBzdPA+mF0FCsBtQTluhdvzesvaUEfns//m47VRak5yDmY3QIKjECYT4ZkVOVY2PFlNwYX7zFK+O9+RGuZCA8vlG0r7yqQzE70IPVHpAPXQS9leTLMlST+BFQ52MmUkrnm3SZETGyFmw+XG8JORASnXZ+NIaBCdneyfp0W5ArZbihaLTs4PA/uuJwWxS6/JE/oGQpJ1iy6xHyMaYYtxx3hwW1CJ1Aya2taARix+IQAPdilJd0mCeS5KN9mX+a2oJKDrgA6NTt7R8Z9rYRv2s993HbHPCEf3c=
   file:
-    - README.md
-    - setup.py
-    - LICENSE
-    - kwikapi/api.py
-    - kwikapi/__init__.py
+  - README.md
+  - setup.py
+  - LICENSE
+  - kwikapi/api.py
+  - kwikapi/__init__.py
   name: kwikapi-0.5.9
   tag_name: 0.5.9
-  on:
+  true:
```

Comment:
This should not be changed I guess @Rajendra-R

Reply:
* We are using python `PyYAML` module it is doing the above conversion but we checked there is no problem with that.